Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift

```diff
@@ -47,7 +47,7 @@ class HistoryPanelViewModel: Loggable, FeatureFlaggable {
     private let profile: Profile
     // Request limit and offset
     private let queryFetchLimit = 100
-    private var currentFetchOffset = 0
+    private(set) var currentFetchOffset = 0
```

Comment:
Based on our discussion earlier about possible misuses of this, I think we can add a comment about this being a test only addition.